Formula/c/caddy.rb

```diff
@@ -50,7 +51,10 @@ def caveats
     keep_alive true
     error_log_path var/"log/caddy.log"
     log_path var/"log/caddy.log"
-    environment_variables XDG_DATA_HOME: "#{HOMEBREW_PREFIX}/var/lib"
+    environment_variables(
+      XDG_DATA_HOME: "#{HOMEBREW_PREFIX}/var/lib",
+      HOME:          "#{HOMEBREW_PREFIX}/var/lib",
+    )
```

Comment:
for service block change, I dont think you need to bump the revision

Reply:
Ok, cool. Removed the revision bump.